Makefile

```
@@ -321,14 +340,14 @@ $(BUILD_DIR)/ldscript.txt: $(BUILD_DIR)/$(SPEC)
 $(BUILD_DIR)/undefined_syms.txt: undefined_syms.txt
 	$(CPP) $(CPPFLAGS) $< > $@
 
-$(BUILD_DIR)/baserom/%.o: baserom/%
+$(BUILD_DIR)/baseroms/$(VERSION)/segments%.o: baseroms/$(VERSION)/segments%
```

VERSION is already part of BUILD_DIR so ideally we'd have

```suggestion
$(BUILD_DIR)/baserom/%.o: baseroms/$(VERSION)/segments/%
```
But the relevant bit in setting O_FILES may become unreadable

This would matter a lot in the spec that iirc currently includes some baserom files, and in the near future many more if we ifdef-build-OK another version this way

---

Oh right, I didn't notice this.
This will be fun to deal with...